example/src/App.js

```diff
@@ -11,19 +11,30 @@ const getItem = index => {
   };
 };
 
-const getItems = n => {
-  return Array(n).fill(null)
-    .map((item, index) => getItem(index));
+const getItems = () => {
+  return fetch("http://localhost:8080/api/process/stages")
```

Comment:
Chyba nie powinno być localhosta

Reply:
Ale to jest tylko example XDDD

Reply:
Example nie example nie każdy musi mieć postawiony serwer w springu by się odpalał na porcie 8080 XD

Reply:
Ale akurat ja miałem, stąd nazwa example XDDDDDD Ponadto we Flasku, fastApi, czy Nodzie też możesz sobie postawić serwer na 8080, nie wiem, w czym problem XDDDD